src/components/table/index.md.njk

```diff
@@ -49,6 +49,8 @@ You can use the [width override classes](../../styles/spacing#width-override-cla
 
 If the [width override classes](../../styles/spacing#width-override-classes) do not meet your needs you can create your own width classes and apply them to the cells in the table head. These can be added using the `classes` option in the Nunjucks macro or adding the class directly to the individual cells within `govuk-table__head` as below.
 
+See [Extending and modifying components in production](https://design-system.service.gov.uk/get-started/extending-and-modifying-components/) for more information.
```

Comment:
This feels clumbsy, is there a better way to call out the guidance? Or maybe it's not necessary to call it out? @amyhupe @soupdragon99

Reply:
I think it looks good, and it's relevant to say there. 

I would probably say "To learn more about this, read guidance on [how to extend and modifying components in production](link}.